content/applications/productivity/iot/config/troubleshooting.rst

```diff
@@ -2,133 +2,150 @@
 Troubleshooting
 ===============
 
-IoT Box Connection
+IoT box connection
 ==================
 
-I can't find the pairing code to connect my IoT Box
----------------------------------------------------
+Unable to locate the pairing code to connect the IoT box
+--------------------------------------------------------
 
-The pairing code should be printed on receipt printers connected to the
-IoT Box and should also be displayed on connected monitors.
+The pairing code should be printed on receipt printers connected to the :abbr:`IoT (Internet of
+Things)` box and should also be displayed on connected monitors.
 
 The pairing code doesn't show under the following circumstances:
 
--  The IoT Box is already connected to an Odoo database;
+- The :abbr:`IoT (Internet of Things)` box is already connected to an Odoo database.
+- The :abbr:`IoT (Internet of Things)` box is not connected to the Internet.
+- The code is only valid for 5 minutes after the :abbr:`IoT (Internet of Things)` box has started.
+  it's automatically removed from connected displays when this time has expired.
+- The version of the :abbr:`IoT (Internet of Things)` box image is too old. If the :abbr:`IoT
+  (Internet of Things)` box image is from an earlier version, then the SD card of the :abbr:`IoT
+  (Internet of Things)` box will need to be re-flashed to update the image (see :doc:`Flashing the
+  SD Card <flash_sdcard>`).
 
--  The IoT Box is not connected to the Internet;
+If none of the cases listed above correct the issue, then make sure that the :abbr:`IoT (Internet of
+Things)` box has correctly started, by checking that a fixed green LED is showing next to the power
+port.
 
--  The code is only valid for 5 minutes after the IoT Box has started. It's
-   automatically removed from connected displays when this delay has expired;
+IoT box is connected but it's not showing in the database
+---------------------------------------------------------
 
--  The version of the IoT Box image is too old. It should use version 20.06
-   or more recent. If your IoT Box image is from an earlier version, then
-   you will have to reflash the SD card of your IoT Box to update the image
-   (see :doc:`Flashing your SD Card <flash_sdcard>`)
+When an :abbr:`IoT (Internet of Things)` box connects to a database, the :abbr:`IoT (Internet of
+Things)` box might restart. If that is the case, it might take up to five minutes before appearing
+in the database. If after some time the :abbr:`IoT (Internet of Things)` box is still not showing,
+make sure that the database can be reached from the :abbr:`IoT (Internet of Things)` box and that
+the server doesn't use a multi-database environment.
```

Comment:
```suggestion
When an :abbr:`IoT (Internet of Things)` box connects to a database, it may restart. If so, it can take 
up to five minutes before appearing in the database. If the :abbr:`IoT (Internet of Things)` box is still not showing after five minutes, make sure that the box can reach the database and that
the server doesn't use a multi-database environment.
```